src/store/calendarObjectInstance.js

```diff
@@ -46,6 +46,20 @@ import {
 import { mapAlarmComponentToAlarmObject } from '../models/alarm.js'
 import { getObjectAtRecurrenceId } from '../utils/calendarObject.js'
 
+/**
+ * @param {Array} entries Object entries
+ *
+ * @returns {Object}
+ */
+function objectFromEntries(entries) {
+	const obj = {}
+	entries.forEach(([k, v]) => {
```

Comment:
Please use full readable variable names. We have a minifier in place for the production code. The source version should always use readable names.

Reply:
I removed the function as it's not necessary anymore.

I know you have minifier and I didn't use k, v to have final js small :) I'm just used to using `k` as key and `v` as value in generic functions instead of `key` and `value`.